core/src/main/java/org/apache/servicecomb/core/Invocation.java

```diff
@@ -188,15 +188,16 @@ public String getMicroserviceQualifiedName() {
   public void triggerStartProcessingEvent() {
     this.startProcessingTime = System.nanoTime();
     EventUtils.triggerEvent(new InvocationStartProcessingEvent(
-        operationMeta.getMicroserviceQualifiedName(), this.invocationType, startProcessingTime - startTime));
+        operationMeta.getMicroserviceQualifiedName(), this.invocationType));
   }
 
-  public void triggerFinishedEvent(int statusCode, boolean success) {
+  public void triggerFinishedEvent(int statusCode) {
     long finishedTime = System.nanoTime();
     EventUtils
-        .triggerEvent(new InvocationFinishedEvent(operationMeta.getMicroserviceQualifiedName(),
-            this.invocationType, finishedTime - startProcessingTime,
-            finishedTime - startTime, statusCode, success));
+        .triggerEvent(new InvocationFinishedEvent(operationMeta.getMicroserviceQualifiedName(), this.invocationType,
+            startProcessingTime - startTime,
```

Comment:
Can you explain the meaning of startTime, startProcessingTime?

Reply:
On producer side , startTime is when invocation received,startProcessingTime is when invocation fetch from queue and start execute;